percy-xcui/Sources/PercyXcui/AppPercy.swift

```diff
@@ -7,7 +7,7 @@ public class AppPercy {
   // swiftlint:disable:next identifier_name
   static var _allowedDevicesMessageShown: Bool = false
 
-  static func setLogLevel(level: String = "info") {
+  public func setLogLevel(level: String = "info") {
```

Comment:
any reason to change it from static to public?

Reply:
static is not required here, this was a bug, we were not able to set loglvl